RELEASING.rst

```diff
@@ -0,0 +1,55 @@
+Releasing sphinxcontrib-images
+==============================
+
+1. Test releasing to TestPyPI
+
+   - Bump the package version by editing ``sphinxcontrib/images.py``
+     (i.e. from 0.9.2 to 0.9.3.rc1 -- use .rcN for subsequent releases),
+     create a release branch with this change in the
+     https://github.com/sphinx-contrib/images repository, and finally create a
+     pull request.
+
+   - Trigger `publishing to TestPyPI via the GitHub UI`_ and select the release
+     branch as where to run the workflow.
+
+2. Test the TestPyPI release
+
+   - Make a new virtual environment::
+
+        read -p 'Version: ' VERSION \
+           && python3 -m venv venv-$VERSION \
+           && . venv-$VERSION/bin/activate
+
+   - Install the (pre-)release (see [1]_) and verify that the correct version
```

Comment:
I think it is a good idea to just use `rcN` in `image.py` - it was confusing with `preN` being normalized to `rcN`. So "(pre-)release" could be changed to "release(-candidate)".